lib/seafoam/passes/truffle.rb

```diff
@@ -2,6 +2,8 @@
 
 require "tsort"
 
+require_relative "truffle_translators/translators"
```

Comment:
`require` searches the `$LOAD_PATH` while `require_relative` searches relative to the current file. Since I'm not trying to load from an external library, `require_relative` seemed most appropriate.